index.js

```diff
@@ -1 +1,124 @@
 // implement your API here
+
+const express = require('express');
+const server = express();
+
+const Users = require('./data/db');
+
+server.use(express.json());
+
+server.get('/', (request, response) => {
+  response.send('Hello from our new server!!!');
+});
+
+//POST
+
+server.post('/api/users', (request, response) => {
+  if (!request.body.name && !request.body.bio) {
+    response.status(400).json({
+      success: false,
+      error: 'Please provide name and bio for the user.'
+    });
+  } else {
+    Users.insert(newUser)
```

Comment:
where is newUser defined? I'm not seeing it anywhere. Needs to be request.body otherwise.